src/app_service/service/community/dto/community.nim

```diff
@@ -29,6 +31,17 @@ type CommunitySettingsDto* = object
 type CommunityAdminSettingsDto* = object
   pinMessageAllMembersEnabled*: bool
 
+type ContractAddressDto* = object
+  chainId*: int
+  address*: string
+
+type CommunityTokensMetadataDto* = object
+  addresses*: seq[ContractAddressDto]
```

Comment:
How about we make this a `Table[int, string]` instead?

Reply:
Done.